test/scaler/scaler_native_test.exs

```diff
@@ -0,0 +1,101 @@
+defmodule Scaler.NativeTest do
+  use ExUnit.Case, async: true
+  alias Membrane.Payload
+  alias Membrane.FFmpeg.SWScale.Scaler.Native, as: Scaler
+
+  defp make_test(input_width, input_height, output_width, output_height) do
```

Comment:
I am not sure about the convension in test files. Most of the time I see helper functions before tests like [in swresample plugin](https://github.com/membraneframework/membrane_ffmpeg_swresample_plugin/blob/master/test/membrane_ffmpeg_swresample_plugin/converter_test.exs) and [h264 plugin](https://github.com/membraneframework/membrane_h264_ffmpeg_plugin/blob/master/test/encoder/encoding_test.exs). In core (e.g. [here](https://github.com/membraneframework/membrane_core/blob/master/test/membrane/core/element_test.exs) and [here](https://github.com/membraneframework/membrane_core/blob/master/test/membrane/core/child/pad_controller_test.exs)) there are even private functions before tests. In source files I know that private functions should be at the end. But do we have some standard regarding test files, or is it arbitrary?

Reply:
Not really. In general, the functions before tests make sense if they perform a setup (often using `setup` macro from ExUnit, they cannot be private then), the helpers common for at least a couple of functions go last and helpers for a specific function (e.g. when doing recursion and having pairs like `foo` and `do_foo`) go right after the said function.
However these are only my suggestions and the way I'm used to, not a strict guideline.

One more note though - here it would make sense to keep this function on top but rename it to something like `scaling_test`